### DISCUSSION TITLE
&Privé

### DISCUSSION THREAD

**Discussion by Weiko07** [iptv-org/community#716] [Q&A]
**_Channel Name: &Privé
**_Channel ID: AndpriveHD.in
**_Country: India
**_Broken Link (from playlist):_** xxx
**_Possible Replacement (optional):_** xxx
**_Notes (optional): Same as &Flix channel it's broken.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19749
